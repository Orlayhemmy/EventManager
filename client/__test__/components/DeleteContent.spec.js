import React from 'react';
import DeleteContent from '../../components/Common/Delete';

describe('Delete Event Component', () => {
  const onDelete = jest.fn();
  const title = '';
  const onCancel = jest.fn();
  const wrapper = shallow(<DeleteContent onCancel={onCancel} onDelete={onDelete} title={title} />)
  ;
  it('should render component without error', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('check the length of the containing div in the component', () => {
    expect(wrapper.find('#delete-content').length).toEqual(1);
  });
});
