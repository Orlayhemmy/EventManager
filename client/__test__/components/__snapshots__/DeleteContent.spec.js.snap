// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete Event Component should render component without error 1`] = `
<div
  id="delete-content"
>
  <p>
    Are you sure you want to delete 
    ?
  </p>
  <i
    className="fa fa-trash red"
    id="yes"
    onClick={[MockFunction]}
  />
  <i
    className="fa fa-save green"
    id="no"
    onClick={[MockFunction]}
  />
  <br />
  <span>
    <br />
    Yes
  </span>
  <span>
    <br />
    No
  </span>
</div>
`;
